.github/workflows/foss-sync.yml

```diff
@@ -1,10 +1,10 @@
 name: Sync PostHog FOSS
 
-on:
-    push:
-        branches:
-            - master
-            - main
+# on:
```

Comment:
Let's add a TODO here outlining what's going on or create an issue. This will get lost otherwise among other changes.

Also worth adding a banner at the top of posthog-foss.

Reply:
done